real_estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+class EstatePropertyTag(models.Model):
```

Comment:
```suggestion
from odoo import fields, models


class EstatePropertyTag(models.Model):
```
Exactly two empty lines, before defining a class